README.md

```diff
@@ -68,6 +68,8 @@ The meaning of each error code is given here:
 * `E1024` - Your [Discord guild](https://discord.com/developers/docs/resources/guild) does not contain a [role](https://discord.com/developers/docs/topics/permissions#role-object) with the name "**@Archivist**".
 (This [role](https://discord.com/developers/docs/topics/permissions#role-object) is required for the `/archive` [command](https://discord.com/developers/docs/interactions/application-commands))
 
+* `E1025` - Your [Discord guild](https://discord.com/developers/docs/resources/guild) does not contain a [role](https://discord.com/developers/docs/topics/permissions#role-object) with the name "**@Applicant**". This [role](https://discord.com/developers/docs/topics/permissions#role-object) is not required but is considered an already inducted role for the sake of introduction reminders. 
```

Comment:
You state that this role is not required, yet an error is raised if it does not exist. Either the documentation must match that the role must exist, or the lack of the role must be ignored rather than logging an error.

If it is the desire for the lack of the role to be ignored, `TeXBot.applicant_role` should return `discord.Role | None`